cloudformation/lecture13_Network.yml

```
@@ -0,0 +1,169 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Description: RaiseTech-lecture13-Network template
+
+Parameters:
+  NameBase:
+    Description: this is base name.
+    Type: String
+    Default: RaiseTech-lecture13
+
+  AvailabilityZone1a:
+    Description: Please input Availability Zone
+    Type: AWS::EC2::AvailabilityZone::Name
+    Default: ap-northeast-1a
+
+  AvailabilityZone1c:
+    Description: Please input Availability Zone
+    Type: AWS::EC2::AvailabilityZone::Name
+    Default: ap-northeast-1c
+
```

上記の命名ですが、1aと1c以外もAZが指定できる可能性があるのなら次のように書いて入力制限してもいいかなと思いました！
```suggestion
    AvailabilityZone1a:
    Type: AWS::EC2::AvailabilityZone::Name
    AllowedValues:
      - ap-northeast-1a
    Default: ap-northeast-1a

  AvailabilityZone1c:
    Type: AWS::EC2::AvailabilityZone::Name
    AllowedValues:
      - ap-northeast-1c
    Default: ap-northeast-1c

```